domains/raahi.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Ravan745",
+        "email": "carelessgaming8@gmail.com"
+    },
+    "record": {
+        "RECORDTYPE": "A"
```

This is not how records work
```suggestion
        "A": "YOUR_IP_HERE"
```
see https://www.is-a.dev/docs/domain-structure/